src/neptune/Utility.py

```diff
@@ -162,7 +161,7 @@ def generateSequence(integer, length):
         elif base == 3:
             sequence = "T" + sequence
 
-        current = current / 4
+        current = current // 4
```

Comment:
Nothing to change. I actually didn't realize `//` does integer division in Python 3 (and Python 2, `/` did integer division by default).

Reply:
Yeah, I didn't expect this one. I think I was assuming Python3 still did integer division like this until things started breaking. Haha.